code/parachain/runtime/common/src/fees.rs

```diff
@@ -48,15 +47,10 @@ pub fn multi_existential_deposits<AssetsRegistry>(_currency_id: &CurrencyId) ->
 }
 
 #[cfg(not(feature = "runtime-benchmarks"))]
-pub fn multi_existential_deposits<
-	AssetsRegistry: AssetRatioInspect<AssetId = CurrencyId>
-		+ AssetExistentialDepositInspect<AssetId = CurrencyId, Balance = Balance>,
->(
+pub fn multi_existential_deposits<AssetsRegistry: AssetRatioInspect<AssetId = CurrencyId>>(
 	currency_id: &CurrencyId,
 ) -> Balance {
-	AssetsRegistry::existential_deposit(*currency_id)
-		.ok()
-		.or_else(|| WellKnownForeignToNativePriceConverter::existential_deposit(*currency_id))
+	PriceConverter::<AssetsRegistry>::to_asset_balance(NATIVE_EXISTENTIAL_DEPOSIT, *currency_id)
```

Comment:
Instead of using the conversion of 1/10 PICA * ratio, would it be wiser to use 1/10 of whatever the asset as the ED for said asset?
@KaiserKarel @vimukthi-git @dzmitry-lahoda 
Reasoning/Context:
> Ratio is pre Dex predictor of price. Later dex will be used. So not sure if ED to be as volatile as DEX.

> Meaning you're concerned that only using ration to calculate ED will be too volatile once the market is active?

> Yes. We use ratio for fees because there is no dex to tell the ratio (or we do not want market price).  For ED it may be default we do not want market price ever. I do not know.

> Yeah, I'd be a bit annoyed if my balance that wasn't previously dust got turned into dust and removed due to a spike in value

> It might make senses to define ED relative to whatever 1 is considered to be in any given asset

Reply:
If you had such a small amount that got turned into dust, I don't think this would matter tbh. But having a more predictable value like TWAP for asset ratio might make sense in the future.

Reply:
dust accounts are in wallet.